testdata/serviceconfig.yml

```diff
@@ -7,6 +7,23 @@ rules:
   aspects:
   - kind: quotas
     params:
+  - kind: metrics
+    adapter: "prometheus"
+    params:
+      metrics:
+      - descriptor: "request_count"
+        value: "1"
```

Comment:
They're identical: both `"1"` and `1` will be interpreted as strings => the `1` will be `"1"` at runtime. The proto field backing `value` is of type `string` because it represents an expression, not a literal value. However, in this case I want the expression literal `"1"`, and `"1"` correctly maps to `int64(1)` at runtime.

There is a difference between a string literal _in_ an expression (`"\"1\""`) and an expression represented as a string.

I'll update to `1` instead of `"1"` so its it looks more natural when a user glances over it (though IMO it's actually worse since it makes it less obvious what's actually happening).